todo_app/data/session_items.py

```diff
@@ -43,9 +43,27 @@ def doneitems(self):
 #loading environment variables for KEY,TOKEN to be used if functions are invoked without flask
 load_dotenv()
 
-
 def getBoardid():
-    return os.environ['TRELLO_BOARD_ID']
+   url_for_board = 'https://api.trello.com/1/members/me/boards'
+
+
+   query = {
+      'key': os.getenv('KEY'),
+      'token': os.getenv('TOKEN')
+   }
+
+   response = requests.request(
+      "GET",
+      url_for_board,
+      params=query
+   )
+   
+   getBoardresponse = response.text
+   getBoardresponse = json.loads(getBoardresponse)
+   return getBoardresponse[0]["id"]
```

Comment:
Why did you make this change?

This means that you don't control which board gets used when you have multiple boards. Most importantly, it means the e2e test can't make the app use a dedicated test board